config-templates/ibm/ibm-object-storage-plugin/2.2/custom-parameters.json

```diff
@@ -29,8 +29,8 @@
         "required": "true"
     },
     {
-        "description": "Enter the object storage service provider. Supported providers are 'IBM', 'AWS' and 'Wasabi'. For other providers, you must explicitly provide the COS endpoint",
-        "displayname": "Object storage service provider",
+        "description": "Enter Object storage service provider. Supported providers are 'IBM', 'AWS' and 'Wasabi'. For other providers, you must explicitly provide the Object storage service endpoint",
```

Comment:
`Object Storage Service Endpoint` we will have this string same as the display name of cos-endpoint